srcs/vm/ops/aff.c

```diff
@@ -6,15 +6,19 @@
 /*   By: merlijn <merlijn@student.codam.nl>           +#+                     */
 /*                                                   +#+                      */
 /*   Created: 2020/08/19 18:05:23 by merlijn       #+#    #+#                 */
-/*   Updated: 2020/09/10 14:27:10 by wmisiedj      ########   odam.nl         */
+/*   Updated: 2020/09/12 01:14:44 by mvan-eng      ########   odam.nl         */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "vm.h"
 
+/*
+**	only counted as OK by tester if it doesn't do anything
```

Comment:
Can it maybe go wrong because it's cast to char? Maybe we should check if .value is actually a printable character e.g. (is_ascii) and only then print?

Reply:
In de diff van de dumps heeft de expected_dump een 0 extra, dat is wat ik heb kunnen vinden tot nu toe